rest-server/src/config/job.js

```diff
@@ -58,6 +58,9 @@ const jobConfigSchema = Joi.object().keys({
       }))
     .min(1)
     .required(),
+  gpuType: Joi.string()
+    .allow('')
+    .default(''),
```

Comment:
>'' [](start = 13, length = 2)

please check with QIngcha about the behavior when gpuType is '' I remember he says empty string will result in non of the gpu type being matched.